lib/qx/tool/cli/commands/Lint.js

```diff
@@ -114,9 +119,14 @@ qx.Class.define("qx.tool.cli.commands.Lint", {
           const formatter = linter.getFormatter(outputFormat);
           const s = formatter(report.results);
           if (this.argv.outputFile) {
-            console.info(this.argv.outputFile);
+            if (this.argv.verbose) {
+              console.info(`Report to be written to $(this.argv.outputFile)`);
+            }
             await fs.writeFileAsync(this.argv.outputFile, s, "UTF-8")
               .catch(e => console.error(`Error writing report to ${this.argv.outputFile}:` + e.message));
+            if (this.argv.verbose) {
```

Comment:
sorry to be so picky - I'd either use try/catch here or wrap the second console info into `then()`, since it will be printed even in the case of a write error.